### ISSUE TITLE
whats the username and password for localhost:8080

### ISSUE THREAD

**Issue by Haotian-Shi-cyber** [johnnykv/heralding#145]

**Comment by Haotian-Shi-cyber** [comment]
yeah im running it using docker. but i cannot catch any ssh login session, im running it using 
docker run -p 80:80 heralding
but it can only log the http credentials

**Comment by realsdx** [comment]
For ssh honeypot , you need to map the port 22 too.
This `docker run -p 80:80 -p 22:22 heralding` log enable you to log HTTP and ssh credentials
